README.md

```diff
@@ -30,6 +30,7 @@
 - ```gcc-aarch64-linux-gnu``` for compiling to ARM
 - ```curl``` to download the kernel
 - ```bsdtar``` for writing the archive file
+- ```f2fs-tools``` for working with the chromebook's eMMC storage
```

Comment:
maybe change it to something along the lines of "for creating filesystem used by cadmium"?
it would describe the purpose more

Reply:
yeah i agree, i wasnt really that confident about my descriptions i wrote up